docs/reference/3-admin-api/whats-new.md

```diff
@@ -3,7 +3,27 @@
 This section provides an overview of what's new in the latest versions of the
 Admin API.
 
-## Updates in Admin API v2.2 (Latest Release)
+## Updates in Admin API v2.3 (Latest Release)
+
+### Features
+
+* Tenant endpoints on V2: new read-only and management endpoints.
```

Comment:
No management of tenants, right? Just read only. Can you list the new endpoint, i.e. `/v2/tenants`